Localize/lang/strings.json

```
@@ -782,6 +780,7 @@
   "_+JCU+Y.comment": "Tooltip for on-prem gateway connection checkbox",
```

"on-premises", not "on-prem":

```suggestion
  "_+JCU+Y.comment": "Tooltip for the on-premises data gateway connection checkbox",
```